src/nodes/TextToSpeechNode.js

```diff
@@ -0,0 +1,20 @@
+import { LiteGraph } from '@comfyorg/litegraph';
+import { colors } from '../configs';
+
+class TextToSpeechNode extends LiteGraph.LGraphNode {
+    constructor() {
+        super();
+        this.title = "Text to Speech";
+        this.addInput("text", "string");
+        this.addOutput("audio", "audio");
+        this.color = colors.yellowDark;
+    }
+
+    onExecute() {
+        const text = this.getInputData(0);
+        // Implement TTS logic here
+        this.setOutputData(0, { type: "audio", data: "synthesized_audio_data" });
```

Comment:
We probably should just delete this file, it duplicates TTSNode.

I guess it was only for temporary test purposes? Or rename it as `TTSNodeTest.js` to make it clear that this is test node?

Reply:
+1 to delete it.  I believe it was simply a consequence of moving fast at a hackathon.

Reply:
Ok @vipyne - I deleted TextToSpeechNode - we can merge this PR if you don't have any other suggestions.

And pls let me know what are the future plans - how can I help :)